### PR TITLE
More samples code cleanup

### PR DESCRIPTION
**Pull request by drwill-ms**

### CODE REVIEW

File: sdks/csharp/fullClientApp.cs [hunk 1]
```diff
@@ -26,70 +26,77 @@ static async Task Main(string[] args)
...
             Console.WriteLine($"Service client created – ready to go");
             // </Authentication_code>
 
             // <Model_code>
             Console.WriteLine();
-            Console.WriteLine($"Upload a model");
-            var typeList = new List<string>();
+            Console.WriteLine("Upload a model");
             string dtdl = File.ReadAllText("SampleModel.json");
-            typeList.Add(dtdl);
```
Comment (drwill-ms) [inline sdks/csharp/fullClientApp.cs:38]: `typeList` didn't sound like a meaningful variable name

File: sdks/csharp/graph_operations_other.cs [hunk 2]
```diff
@@ -1,20 +1,22 @@
...
+await client.CreateOrReplaceRelationshipAsync("mySourceTwin", "rel001", rel);
 // </CreateRelationship_short>
 
 // ------------------ CREATE RELATIONSHIP (Alternate) ---------------------
 // <CreateRelationship_3>
 // Create Twins, using functions similar to the previous sample
-await CreateRoom("Cafe", 70, 66);
-await CreateFloor("GroundFloor", averageTemperature=70);
+await CreateRoomAsync("Cafe", 70, 66);
+await CreateFloorAsync("GroundFloor", averageTemperature = 70);
```
Comment (drwill-ms) [inline sdks/csharp/graph_operations_other.cs:19]: I'm not sure what `averageTemperate = 70` means as a parameter. This feels like nonsense to me, but as there is no impl of `CreateFloorAsync`, I don't know how to interpret it.

File: sdks/csharp/graph_operations_sample.cs [hunk 3]
```diff
@@ -1,64 +1,60 @@
...
+            // Create the twin
             await client.CreateOrReplaceDigitalTwinAsync<BasicDigitalTwin>(srcId, floorTwin);
             Console.WriteLine("Twin created successfully");
 
-            //Create second (Room) digital twin
-            BasicDigitalTwin roomTwin = new BasicDigitalTwin();
+            // Create second (Room) digital twin
+            var roomTwin = new BasicDigitalTwin();
             string targetId = "myRoomID";
-            roomTwin.Metadata = new DigitalTwinMetadata();
```
Comment (drwill-ms) [inline sdks/csharp/graph_operations_sample.cs:49]: The type already has an object allocated, so creating a new one is not necessary

File: sdks/csharp/graph_operations_other.cs (hunk 2 above)
Comment (alexkarcher-msft) [reply sdks/csharp/graph_operations_other.cs:19]: Good catch! It's from [the graph concept doc](https://docs.microsoft.com/en-us/azure/digital-twins/concepts-twins-graph#create-relationships) and it doesn't really make sense there either. 

@baanders any reason we can't use the CreateRelationship_short example from above here as well? That one is called in [this doc](https://docs.microsoft.com/en-us/azure/digital-twins/how-to-use-apis-sdks#create-a-relationship)

We should probably add the try/catch block to both examples regardless, right?

**Review by alexkarcher-msft — CHANGES_REQUESTED** [review]
This looks awesome! I added some comments around that strange relationship example that hopefully clear everything up

File: sdks/csharp/fullClientApp_excerpt_model.cs [hunk 4]
```diff
@@ -1,7 +1,7 @@
 Console.WriteLine();
 Console.WriteLine($"Upload a model");
-var typeList = new List<string>();
 string dtdl = File.ReadAllText("SampleModel.json");
-typeList.Add(dtdl);
+var models = new List<string>();
```
Comment (azabbasi) [inline sdks/csharp/fullClientApp_excerpt_model.cs:4]: You can instantiate the list along with the dtld like you did earlier in this PR

File: sdks/csharp/graph_operations_other.cs [hunk 5]
```diff
@@ -1,20 +1,22 @@
...
-rel.TargetId = "myTargetTwin";
-rel.Name = "contains"; // a relationship with this name must be defined in the model
+var rel = new BasicRelationship
+{
+    TargetId = "myTargetTwin",
+    Name = "contains", // a relationship with this name must be defined in the model
+};
 // Initialize properties
-Dictionary<string, object> props = new Dictionary<string, object>();
+var props = new Dictionary<string, object>();
```
Comment (azabbasi) [inline sdks/csharp/graph_operations_other.cs:9]: Same here, we can instantiate and add to the dictionary inline

**Review by azabbasi — APPROVED** [review]

File: sdks/csharp/graph_operations_other.cs (hunk 2 above)
Comment (baanders) [reply sdks/csharp/graph_operations_other.cs:19]: @alexkarcher-msft Sounds good to me--this doesn't seem to add much value as an alternate example (especially if we add the try/catch block to the other one as well), and the use of methods with no implementation could just be confusing.

I'm ok with removing this example and updating the doc to use CreateRelationship_short in its place.

Comment (baanders) [reply sdks/csharp/graph_operations_other.cs:19]: Switching out the example in the doc in PR [#145725](https://github.com/MicrosoftDocs/azure-docs-pr/pull/145725), which should go live at 3 PM PST today (2/4). After that, it's safe to remove CreateRelationship_3.

**Review by alexkarcher-msft — APPROVED** [review]
Love it! Ship it! 🚢🚢🚢

**Review by baanders — APPROVED** [review]

**Pull request merged**